test/projectManager.ts

```diff
@@ -181,25 +257,25 @@ export class ProjectManager {
     }
 
     /**
-     * Executes a child process and logs its output to the console.
+     * Executes a child process and logs its output to the console and returns its output in the promise as a string
      */
-    public static execAndLogChildProcess(command: string, options?: child_process.IExecOptions): Q.Promise<void> {
-        var deferred = Q.defer<void>();
+    public static execAndLogChildProcess(command: string, options?: child_process.IExecOptions, output: boolean = true): Q.Promise<string> {
```

Comment:
If the logging behaviour is now toggle-able, would it make sense to just name this `execChildProcess`?